.github/workflows/integration.yml

```diff
@@ -59,13 +59,15 @@ jobs:
         working-directory: go
         run: go install ./cmd/berty-integration
       - name: Run integration tests with production config
+        id: run
         run: |
           berty-integration \
             -integration.testbot=`cat ./config/config.gen.json | jq -r '.berty.contacts["testbot"].link'`
+          echo "::set-output name=runUrl::https://github.com/${GITHUB_REPOSITORY}/actions/runs/${GITHUB_RUN_ID}"
```

Comment:
move this to a dedicated job; the "Run integration" is the most important rule of this workflow and should not be polluted